Makefile

```diff
@@ -369,6 +369,7 @@ install_prerequisites:
 	[ -z "$(shell which python)" -a -n "$(shell which python3)" ] && sudo apt-get install -y -q python-is-python3 || true
 	sudo apt-get install -y -q clang-tidy || true
 	sudo apt-get install -y -q clang clang-format ragel
+	sudo apt-get install -y -q llvm || true
```

Comment:
Is this package really a necessary prerequisite?

Reply:
It is used to get llvm-addr2line. gnu addr2line sometimes cannot get line info for some addresss, while llvm-addr2line doesn't have this problem, I doubt it's because the kernel is compiled by clang, but I don't have evidence for that.